library/src/main/java/io/andref/rx/widgets/ListViewCard.java

```diff
@@ -267,14 +153,14 @@ public void setItems(List<Item> items)
         return mButtonClicks;
     }
 
-    public Observable<Item> iconClicks()
+    public Observable<Pair<Item, Integer>> iconClicks()
     {
-        return mIconClicks;
+        return mListViewCardAdapter.iconClicks();
     }
 
-    public Observable<Item> itemClicks()
+    public Observable<Pair<Item, Integer>> itemClicks()
```

Comment:
I'm not sold on changing the upstream API for something we're not actually going to use. Plus, it adds a `Pair`, which sucks a giant banana. I think the fact that you added comparator methods alleviates the need to have the position at this time.